ddtrace/version.py

```diff
@@ -1,13 +1,23 @@
-def get_version() -> str:
-    try:
-        from ._version import version
-
-        return version
-    except ImportError:
-        from importlib.metadata import version as ilm_version
-
-        try:
-            return ilm_version("ddtrace")
-        except ModuleNotFoundError:
-            # package is not installed
-            return "dev"
+__all__ = [
+    "__version__",
+    "__version_tuple__",
+    "version",
+    "version_tuple",
+]
+
+TYPE_CHECKING = False
+if TYPE_CHECKING:
+    from typing import Tuple
+    from typing import Union
+
+    VERSION_TUPLE = Tuple[Union[int, str], ...]
+else:
+    VERSION_TUPLE = object
+
+version: str
+__version__: str
+__version_tuple__: VERSION_TUPLE
+version_tuple: VERSION_TUPLE
+
+__version__ = version = "4.1.0.dev0"
+__version_tuple__ = version_tuple = (4, 1, 0, "dev0")
```

Comment:
This should be derived from `__version__` to eliminate the possibility of coding errors where `__version__` and `__version_tuple__` are different.